docs/src/man/comparisons.md

```diff
@@ -93,6 +93,20 @@ examples above do not synchronize the column names between pandas and DataFrames
 (you can pass `renamecols=false` keyword argument to `select`, `transform` and
 `combine` functions to retain old column names).
 
+### Mutating operations
+
+| Operation          | pandas                                                | DataFrames.jl                                |
+| :----------------- | :---------------------------------------------------- | :------------------------------------------- |
+| Add new columns    | `df['z1'] = df['z'] + 1`                              | `df.z1 = df.z .+ 1`                          |
+|                    |                                                       | `transform!(df, :z => (x -> x .+ 1) => :z1)` |
```

Comment:
maybe add another row showing how to use `insertcols!`, e.g. `insertcols!(df, 2, :const=1)` to insert a column `:const` as a second column of a data frame that consists of `1`s.

Reply:
Sure, updated PR.